### PR TITLE
Update packages.json

### PR DESCRIPTION
**Pull request by 0xTim**
Add latest packages in

### CODE REVIEW

File: packages.json [hunk 1]
```diff
@@ -99,7 +99,37 @@
...
+      "excludedProducts": [],
+      "excludedTargets": [],
+    },
+    {
+      "url": "https://github.com/GraphQLSwift/Graphiti",
+      "excludedProducts": [],
+      "excludedTargets": [],
+    },
+    {
+      "url": "https://github.com/apple/swift-distributed-actors",
```
Comment (yim-lee) [inline packages.json]: This is already in the list.

File: packages.json [hunk 2]
```diff
@@ -99,7 +99,37 @@
...
+      "excludedProducts": [],
+      "excludedTargets": [],
+    },
+    {
+      "url": "https://github.com/apple/swift-cassandra-client",
```
Comment (yim-lee) [inline packages.json]: This is already in the list.

Comment (0xTim) [reply packages.json]: Ahh we're sorting by org

**Review by ktoso — APPROVED** [review]

**Comment by adam-fowler** [comment]
- https://github.com/apple/swift-distributed-tracing hasn't been approved but if we wait another 2 weeks, it'll probably be
- Swift Prometheus has moved. Meant to bring that up in the meeting but forgot

**Comment by 0xTim** [comment]
Updated Swift Prometheus

We can do another PR for tracing and the other packages in the incubation list

**Pull request merged**